### ISSUE TITLE
allele-specific expression

### ISSUE THREAD

**Issue by afadda1** [pachterlab/kallisto#335]
Hi. i have provided the two different alleles in my reference sequence. however, i'm not sure if kallisto can handle this properly. also, what happens in the case of homozygosity? is the tpm reported for each allele is the tpm for the gene (transcripts from the 2 chromosomes)?

**Comment by Yenaled** [comment]
Yes, kallisto can do allele-specific expression; just put the two transcripts (one for each of the two alleles) in the transcriptome index. You'll get a TPM value for each transcript. If only one allele is expressed (as in the case of homozygosity), the other allele will have a much smaller TPM (its TPM may not necessarily be zero though because the abundances are assigned to the alleles based on a probablistic framework). The gene-level TPM should be obtained by summing the TPMs or all transcripts (including alleles) associated with that gene.

**Comment by afadda1** [comment]
thanks a lot for the explanation. very helpful!
